flumine/backtest/backtest.py

```diff
@@ -94,8 +94,10 @@ def _process_market_books(self, event: events.MarketBookEvent) -> None:
 
             # process current orders
             blotter = market.blotter
-            for order in blotter:
+            for order in blotter.live_orders:
```

Comment:
Do we move this to live trading as well?

Reply:
What does this mean?

Reply:
As in add it [here](https://github.com/liampauling/flumine/blob/master/flumine/baseflumine.py#L179) as well